### PR TITLE
Part one answer

### PR DESCRIPTION
**Pull request by carryegaulin31**

### CODE REVIEW

**Comment by Kayla-SA-W** [comment]
Not sure if you know this but I am getting errors when running your program.

Grade: 100 🚀

**Comment by carryegaulin31** [comment]
When I submitted I checked and triple checked all routes and tests and
everything worked. 🤦‍♀️

**Comment by carryegaulin31** [comment]
I'll go back and check

**Comment by Kayla-SA-W** [comment]
@carryegaulin31 it might be from me not running your .sql file that the program itself doesnt run. I'll look at it later when I grade part two

**Comment by carryegaulin31** [comment]
I just looked at it again and it all was working over here again….I can do another push just in case.  I am a little bummed that I had it better but had to dumb it down a bit only for the fact we were on break and wasn’t much help when I got stumped….you’ll most definitely notice that within the commits.    Sent from Mail for Windows 10 From: Kayla-SA-WSent: Sunday, January 3, 2021 1:06 PMTo: carryegaulin31/garbage-pail-apiCc: Carrye; MentionSubject: Re: [carryegaulin31/garbage-pail-api] Part one answer (#2) @carryegaulin31 it might be from me not running your .sql file that the program itself doesnt run. I'll look at it later when I grade part two—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

File: migrations/20201222024742-unnamed-migration.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      createdAt: { type: Sequelize.DATE, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP') },
+      updatedAt: {
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP')
+      },
+      deletedAt: { type: Sequelize.DATE },
+    })
+
+    await queryInterface.createTable('cardASeriesLinkings', {
+      cardListAId: { type: Sequelize.INTEGER, references: { model: 'cardListA', key: 'id' } },
```
Comment (Kayla-SA-W) [inline migrations/20201222024742-unnamed-migration.js:39]: _Nit_ misspelling on the model name

File: index.js [hunk 2]
```diff
@@ -11,22 +14,17 @@ app.get('/', (request, response) => {
...
+app.get('/series', getAllSeriesData)
 app.post('/garbagePail', bodyParser.json(), saveNewKid)
-
-
-
-
-app.all('*', (request, response) => {
-  return response.sendStatus(404)
-})
+app.delete('/garbagePail', deleteAKid)
```
Comment (Kayla-SA-W) [inline index.js:23]: _Nit_ this path doesnt work

**Comment by Kayla-SA-W** [comment]
Great Job 🚀 

-14 delete route doesnt work, misspelling of the model name prevents program from running.

Grade: 86
